common/element_handle.go

```diff
@@ -982,28 +982,31 @@ func (h *ElementHandle) OwnerFrame() api.Frame {
 	}
 	res, err := h.evalWithScript(h.ctx, opts, fn)
 	if err != nil {
-		k6ext.Panic(h.ctx, "getting document element: %w", err)
+		err = fmt.Errorf("getting document element: %w", err)
 	}
-	if res == nil {
-		return nil
+	if res == nil || err != nil {
+		return nil, err
```

Comment:
I find this part a little bit confusing. Would it be better to be more explicit, such as:
```go
        res, err := h.evalWithScript(h.ctx, opts, fn)
	if err != nil {
		return nil, fmt.Errorf("getting document element: %w", err)
	}
	if res == nil {
		return nil, nil
	}
```
?

Reply:
> Thanks for working on this! Seems like we still have things to learn about Goja 😄

Yeah 😄 Thanks!

> I find this part a little bit confusing.

Explicitified here: https://github.com/grafana/xk6-browser/pull/826/commits/d4aec1fafd7cb909d761a5ba54bb54b5339214ee